### ISSUE TITLE
Apply patch for the fuzzy strings when updating template

### ISSUE THREAD

**Issue by capiscuas** [capiscuas/pootle#2]
More info http://bugs.locamotion.org/show_bug.cgi?id=2030

**Comment by capiscuas** [comment]
FUZZY_MATCH_MAX_LENGTH = 4000 fixed it

**Comment by BorislavZlatanov** [comment]
This fixed most of the problem, but some translated strings are still lost.

Here are strings that were lost:

"testing." -Thorstein Veblen

Document Structure TESTING:

testing.

[2] <i>testing</i>

All can be seen here: http://pootle.linguisticteam.org/af/ignore_me/translate.html#filter=all

**Comment by capiscuas** [comment]
what were the original strigs of those lines before they got replaced?
can you attach both original and new .po/pot files so i can make a test?

**Comment by BorislavZlatanov** [comment]
Yeah. Seems like github doesn't accept neither .pot nor .zip/.rar, so I uploaded them here: http://uploading.com/me64fdac/1-Preface-zip

Just click on "Download for free".
